test/perf/calib/test_cameracalibration.py

```diff
@@ -8,6 +8,7 @@
  @brief Performance tests for CameraCalibration class
  @details Performance tests generated using PyTest
 """
+from pickletools import uint8
```

Comment:
Don't think this should be an import. Probably accidently added automatically in vscode when using the numpy uint8 type.